### PR TITLE
tencent-lemon 5.1.9

### PR DESCRIPTION
**Pull request by zuisong**
Created with `brew bump-cask-pr`.

### CODE REVIEW

File: Casks/t/tencent-lemon.rb [hunk 1]
```diff
@@ -1,18 +1,13 @@
 cask "tencent-lemon" do
-  version "5.1.7"
-  sha256 "31bbd9af3b030984f0ead4ac0c7cdcea7f2eb1afabf772e22834a40798900a8d"
+  version "5.1.9"
```
Comment (bevanjkay) [inline Casks/t/tencent-lemon.rb:2]: The upstream website is pointing to version `5.1.8` as found with the previous livecheck.
What is the reason for the change?

Comment (zuisong) [reply Casks/t/tencent-lemon.rb:2]: GitHub and website are both official distribution channels, the official website updates may be delayed. And choosing GitHub makes writing livecheck a bit easier.

Comment (zuisong) [reply Casks/t/tencent-lemon.rb:2]: see https://support.qq.com/products/36664/faqs-more/?id=155762

**Review by bevanjkay — APPROVED** [review]
